### DISCUSSION TITLE
关于下载时产生的临时文件

### DISCUSSION THREAD

**Discussion by NLick47** [yaobiao131/downkyicore#153] [Announcements]
![EL 7IY 1SZ3(BH(30TH( MH](https://github.com/user-attachments/assets/8feaedf1-d200-422a-80bf-4759a5b4b4b7)
大概下载了二十多个视频产生的临时文件。考虑更改为下载完成时清理？或者比如弹幕这种尽量不去创建临时文件，而在内存中处理呢

**Comment by yaobiao131** [comment]
这些是页面缓存，并且数据库里面还有一份，最初我是根据原wpf版本一致的逻辑，可能这个缓存也没有多少用处，但是内存处理要关注内存占用量。可以考虑是否修改，后续优化下数据库储存相关代码，只是最近没有什么时间
